modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/util/repeatable.es.js

```diff
@@ -16,6 +16,10 @@ const FIELD_NAME_REGEX = /(_\w+_)ddm\$\$(.+)\$(\w+)\$(\d+)\$\$(\w+)/;
 
 const NESTED_FIELD_NAME_REGEX = /(_\w+_)ddm\$\$(.+)\$(\w+)\$(\d+)#(.+)\$(\w+)\$(\d+)\$\$(\w+)/;
 
+export const generateInstanceId = Math.random()
+	.toString(36)
+	.substr(2, 8);
```

Comment:
hi, @marcelabc , what do you think about changing "generateInstanceId" from a properties to a method ?
eg.
``` js
export const generateInstanceId = () => Math.random()
	.toString(36)
	.substr(2, 8);
```